core-bundle/src/Controller/AbstractFragmentController.php

```diff
@@ -85,7 +87,11 @@ protected function createTemplate(Model $model, string $templateName): Template
         }
 
         if ($model->customTpl) {
-            $templateName = $model->customTpl;
+            $request = $this->get('request_stack')->getCurrentRequest();
+
+            if (null === $request || !$this->get('contao.routing.scope_matcher')->isBackendRequest($request)) {
```

Comment:
How about adding a comment here that explains why we are not using the custom template in the back end?

Reply:
👍 

Should I add one in `\Contao\ContentElement` as well? Since the same code is used there.